src/org/omegat/core/tagvalidation/TagValidation.java

```diff
@@ -257,7 +257,9 @@ protected static void inspectOrderedTags(List<Tag> srcTags, List<Tag> locTags,
                 }
                 break;
             case SINGLE:
-                // Ignore
+                break;
```

Comment:
It is better to leave an original comment
```suggestion
                // Ignore
                break;
```